assets/js/components/SapSystemLink.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { screen, render, fireEvent as userEvent } from '@testing-library/react';
```

Comment:
Man! What is this? hehe
https://testing-library.com/docs/ecosystem-user-event/

Reply:
oh god, vscode trying to be smart, sorry about that

Reply:
(and the person driving it :sweat_smile: )